<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 189, 182);
border-top-color: rgb(136, 138, 133);
border-color: rgb(46, 52, 54);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Carnet">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
gridline-color: rgb(46, 52, 54);
border-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ingrese su n√∫mero de carnet:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Titulo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>252</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sistema de Monitoreo 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TextBox2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextBox3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Boton</sender>
   <signal>clicked()</signal>
   <receiver>TextBox2</receiver>
   <slot>backward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
